prog/kubernetes/kubernetes_cluster_nexus.rb

```diff
@@ -13,6 +13,12 @@ def self.assemble(name:, kubernetes_version:, private_subnet_id:, project_id:, l
         fail "Invalid Kubernetes Version"
       end
 
+      subnet = Prog::Vnet::SubnetNexus.assemble(
+        Config.kubernetes_service_project_id,
+        location: location,
+        ipv4_range: "10.10.0.0/16" # TODO: a better way?
```

Comment:
don't we have a deconfliction code of some sort? this guarantees any two k8s clusters can never peer via IPv**4** successfully (which, as you know, would be fine with me personally, but probably is not commercially ideal).

Reply:
Summoning @furkansahin superpowers!

Maybe we can reuse `Prog::Vnet::SubnetNexus.random_private_ipv4` with some range customization modification?

Reply:
Something like that, you'd have to filter against the customer's project, rather than within the project implied by the subnet, that may require some refactoring. But I think we can use the same routine.